commcare_connect/opportunity/tests/test_views.py

```
@@ -1761,3 +1761,34 @@ def test_payment_delete_view(client: Client, opportunity: Opportunity, org_user_
 
     response = client.post(url)
     assert response.status_code == 404
+
+
+@pytest.mark.django_db
+def test_visit_export_count_boundary_dates(
+    organization: Organization, org_user_member: User, opportunity: Opportunity, mobile_user: User, client: Client
```

Ruff (0.15.2)

[warning] Unused function argument: `mobile_user`

(ARG001)